worker/jobTypes/githubJob.js

```diff
@@ -40,7 +42,66 @@ class GitHubJobClass {
         }
         return false;
     }
+    
+    async writeEnvFile(isProdDeployJob){
+      //download published branches file to retrieve prefix and check if repo is versioned 
+      const repoObject = {
+        repoOwner: this.currentJob.payload.repoOwner, repoName: this.currentJob.payload.repoName,
+      };
+      const repoContent = await workerUtils.getRepoPublishedBranches(repoObject)
+      var pathPrefix = repoContent.content.prefix
+
+      if(isProdDeployJob){
```

Comment:
I would pull this out into a function -- get the path prefix from a compartmentalized function.